.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -2,35 +2,33 @@ name: Babelfish for PostgreSQL Extensions Bug Report
 description: File a bug report for Babelfish Extensions
 title: "[Bug]: "
 labels: ["bug", "triage"]
-assignees:
-  - amazon-auto
 body:
   - type: markdown
     attributes:
       value: |
-        Thanks for taking the time to fill out this bug report!
+        Thank you for taking the time to fill out this bug report.
   - type: input
     id: contact
     attributes:
       label: Contact Details
-      description: How can we get in touch with you if we need more info?
+      description: Please provide an email address where we can contact you for more information.
       placeholder: ex. email@example.com
     validations:
       required: false
   - type: textarea
     id: what-happened
     attributes:
       label: What happened?
-      description: Also tell us, what did you expect to happen?
-      placeholder: Tell us what you see!
+      description: Please provide details about the bug, and the steps needed to recreate the bug.
+      placeholder: Tell us about the bug.
       value: "A bug happened!"
```

Comment:
I think we discussed removing the 'value' content, but if I misunderstood, can we replace 'A bug happened!' with:

Tell us about the bug.

I'd rather have that line repeated twice than sound too excited about a bug...